source/common/upstream/upstream_impl.h

```diff
@@ -1283,6 +1288,8 @@ class ClusterImplBase : public Cluster, protected Logger::Loggable<Logger::Id::u
   const bool local_cluster_;
   Config::ConstMetadataSharedPoolSharedPtr const_metadata_shared_pool_;
   Common::CallbackHandlePtr priority_update_cb_;
+  UnitFloat drop_overload_{0};
```

Comment:
As this is non-optional, I suggest clarifying the parsing code to always set the `drop_overload_` value.

Reply:
We initialize drop_overload_ to zero here. Why we need to always set it again in parsing code?
Parsing code should only set it if a valid non-zero value is configured.

Reply:
Done